apps/expo/tsconfig.json

```diff
@@ -4,5 +4,5 @@
     "strict": true,
     "jsx": "react-native"
   },
-  "include": ["src", "index.ts", "app.config.ts"]
+  "include": ["app", "utils", "types", "index.ts", "app.config.ts"]
```

Comment:
can we not use src with expo-router 👀 👀

Reply:
We technically can, but we cannot use src + app dir (expo router looks for app dir to find pages)
That's why I thought that having both app dir and src dir just for types and utils would be kind of weird

Reply:
Also, I have totally no clue how changing files in expo app makes NextJS's lint fail haha

Reply:
> We technically can, but we cannot use src + app dir (expo router looks for app dir to find pages)
> 
> That's why I thought that having both app dir and src dir just for types and utils would be kind of weird

You should open an issue about that on their part

Reply:
> > We technically can, but we cannot use src + app dir (expo router looks for app dir to find pages)
> > That's why I thought that having both app dir and src dir just for types and utils would be kind of weird
> 
> You should open an issue about that on their part

Yeah, I definitely will

Reply:
Issue is open: https://github.com/expo/router/issues/231